### PR TITLE
Phantom app : Bugfix - Phantom app bug fixations [PAPP-21922]

### PR DESCRIPTION
**Pull request by hsrivastava-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+tox.ini
```
Comment (paulnawat) [inline .gitignore:1]: @rgil-splunk I believe `tox.ini` should be uploaded, right?
Please see [MaxMind example](https://github.com/splunk-soar-connectors/maxmind/blob/next/tox.ini)

File: phantom.json [hunk 2]
```diff
@@ -1211,8 +1211,7 @@
                 "run_automation": {
                     "description": "Run automation on newly created artifact(s) (default: true)",
```
Comment (paulnawat) [inline phantom.json:1212]: Now that we remove `"default": true`, should we update this description as well?

File: phantom_connector.py [hunk 3]
```diff
@@ -472,14 +478,15 @@ def _tag_artifact(self, param):
...
             self.save_progress("Unable to modify artifact")
             msg = PHANTOM_ERR_UPDATE_ARTIFACT.format(action_result.get_message())
             if not resp_label:
                 msg = "{}. {}".format("The reason of the failure can be the unavailability of the label in the provided artifact", msg)
             return action_result.set_status(phantom.APP_ERROR, msg)
 
-        action_result.set_summary({'tags_added': ', '.join((list(add_tags - tags_already_added))), 'tags_removed': ', '.join((list(remove_tags - tags_already_removed))),
+        action_result.set_summary({'tags_added': ', '.join((list(add_tags - tags_already_added))),
+        'tags_removed': ', '.join((list(remove_tags - tags_already_removed))),
          'tags_already_present': ', '.join((list(tags_already_added))), 'tags_already_absent': ', '.join((list(tags_already_removed)))})
```
Comment (paulnawat) [inline phantom_connector.py:490]: nit: Can we improve the indentation here? It looks inconsistent here, right?

**Review by paulnawat — CHANGES_REQUESTED** [review]
Changes look great. I left a few comments / suggestions.
Could you also update the description of this PR (i.e. fill in the PR form)?
Could you also look into the failed Integration Tests in the pipeline?

File: .gitignore (hunk 1 above)
Comment (hsrivastava-crest) [reply .gitignore]: I have made the suggested code changes.

File: phantom.json (hunk 2 above)
Comment (hsrivastava-crest) [reply phantom.json]: I have made the suggested code changes.

File: phantom_connector.py (hunk 3 above)
Comment (hsrivastava-crest) [reply phantom_connector.py]: I have made the suggested code changes.

**Comment by paulnawat** [comment]
@hsrivastava-crest Thank you for making the change. As mentioned, could you help check why the Integration Tests is failing here? Is it related to your change? If it's not, can we track this failure in a separate ticket?

**Comment by hsrivastava-crest** [comment]
Hi @paulnawat,
Integration tests passed when we are running it separately on Gitlab: https://cd.splunkdev.com/phantom-apps/app-tests/-/pipelines/4217771
Yes the failure which is occurring is due to my change and I have raised an MR for this in Gitlab:

Ticket ID: https://splunk.atlassian.net/browse/PAPP-23128
Pytest MR link: https://cd.splunkdev.com/phantom-apps/app-tests/-/merge_requests/649

**Review by paulnawat — APPROVED** [review]

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**